build/package/.atlascli.goreleaser.yml

```diff
@@ -49,9 +50,6 @@ archives:
       - LICENSE
       - LICENSING-NOTES.txt
       - third_party_notices/**/*
-    replacements:
-      darwin: macos
-      amd64: x86_64
```

Comment:
`replacements` is been deprecated in favour of `file_name_template`. https://goreleaser.com/deprecations/?h=replacements